cmd/controller-manager/main.go

```diff
@@ -161,8 +161,20 @@ func main() {
 		options = append(options, informers.WithNamespace(ns))
 		kubeoptions = append(kubeoptions, kubeinformers.WithNamespace(ns))
 	}
+
+	tweakListOptionsFunc := func(options *metav1.ListOptions) {
+		if len(options.LabelSelector) > 0 {
+			options.LabelSelector += ",app.kubernetes.io/managed-by=tidb-operator"
+		} else {
+			options.LabelSelector += "app.kubernetes.io/managed-by=tidb-operator"
```

Comment:
```suggestion
			options.LabelSelector = "app.kubernetes.io/managed-by=tidb-operator"
```
I think it's more comprehensible.